lib/arc/file.ex

```diff
@@ -20,6 +20,12 @@ defmodule Arc.File do
     end
   end
 
+  # Accepts a data in base64
+  def new(%{filename: filename, base64: base64}) do
+    {:ok, binary} = Base.decode64(base64)
```

Comment:
I believe that it is interesting to add the option `ignore: :whitespace`
https://github.com/elixir-lang/elixir/issues/4186